### PR TITLE
Fix reclaim invites using cEUR

### LINKED ISSUE #907: “Withdrawal from escrow failed”  error message displayed when user restore account with same phone number & account key on “Connect Contacts” screen
**Frequency:** 100% 
**Repro on build version:** Android internal build V1.16.2,  Test Flight Build V1.16.2(68)
App Store Build V1.16.2
**Does not repro build version:** Play store build V1.15.2
**Repro On Devices:** iPhone SE (14.2), iPhone 6+(12.4.5), Vivo V11 pro (10), Redmi Note 8 (9.0), Redmi Note 8 (10.0), iPhone SE 1st Generation (14.6)
**Testing account:** (515) 808 5659  / [Backup Key](https://docs.google.com/spreadsheets/d/1gXY1I4PBkT9f1uB3zxWgUUJJ1zjVeJcqcWLjiIls1f0/edit#gid=0&range=B73)
**Pre-condition:** Make sure that user should have verified account key with same phone  number (which was verified earlier)
**Repro Steps:**
1. Launch the application & complete onboarding education flow
2. Tap on “Restore My Account” button
3. Paste verified account key & enter same number 
4. Now user should on Confirming screen with timer
5. Wait for some time & user redirects to “Connect Contacts” screen
6. Again wait for some time & observed that, 
**Current Behavior:**  “Withdrawal from escrow failed” red error banner displayed after successfully restoring account.
**Expected Behavior:**  User should able to restore account without any error banner
**Impact:** Bad user experience
**Attachment:** [WithdrawFromEscrowFailedErrorMessage.mp4](https://drive.google.com/file/d/1U3afvGSjvHxGJgGYMCgXaEbVr6FzwUFC/view?usp=sharing)

### LINKED ISSUE #908: Valora cEUR Invites Showing a Large Number of Significant Digits
### Issue

Sending an invite with cEUR to a new user shows a large amount of significant digits.

### Current behavior

When a user receives an invite a large number of significant digits are displayed.

### Desired behavior

Users receiving an invite should only see hundredths or thousandths e.g. '1.01' or '1.001'

### Steps to Reproduce

- Send a user an invite to use Valora with an '.5' cEUR
- Observe SMS on recipient phone

### Versions

v1.16.2

### Screenshots

Screenshot_20210719-214135

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Fix the invite flow for cEUR. This PR requires the changes in Komenci of this PR: https://github.com/celo-org/komenci/pull/214 it's deployed in alfajores so far.

### Other changes

N/A

### Tested

Manually

### How others should test

- Invite a new phone number using cEUR.
- See that the invite message has only two decimals and says cEUR.
- Create a new account and verify using the phone number invited.
- See that you get the funds that you had sent before.

### Related issues

- Fixes #908
- Fixes #907

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/invite/saga.ts [hunk 1]
```diff
@@ -86,7 +86,9 @@ export function* sendInvite(
       ? 'sendFlow7:inviteWithEscrowedPayment'
       : 'sendFlow7:inviteWithoutPayment'
     const message = i18n.t(messageProp, {
-      amount: amount?.toString(),
+      amount: amount?.toFixed(2),
```
Comment (tarikbellamine) [inline packages/mobile/src/invite/saga.ts:89]: Why the optional chaining on `amount`? I notice it was there before but it looks like a required param so maybe we should remove it

File: packages/mobile/locales/en-US/inviteFlow11.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
   "redeemInviteTimeout": "Redeem Invite Timed out",
   "accountSetupFailed": "Failed to create your wallet",
   "emptyInviteCode": "Invite code has no funds",
-  "transferDollarsToAccount": "Dollars transferred to your account!",
+  "transferDollarsToAccount": "Invite funds transferred to your account!",
```
Comment (tarikbellamine) [inline packages/mobile/locales/en-US/inviteFlow11.json:16]: I think this is kinda awkward phrasing for a success message. Maybe get a designer's opinion?

**Review by tarikbellamine — APPROVED** [review]
Nice!

**Review by jeanregisser — APPROVED** [review]
🚀

**Comment by ValoraQA** [comment]
Hi @gnardini  We have verified this issue on Latest Android internal build V1.18.0 and updated the status in test quality tool.

Test quality tool link:

Android (11):
https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469324

Verified on Devices: Google Pixel 4a(11.0)

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.18.0 and updated the status in test quality tool.

Test quality tool link:

Android (10):
https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469313

**Verified on Devices:** Redmi Note (10.0)
CC: @gnardini 
Thanks..!!

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Testflight alfajores build V1.18.0 and updated the status in test quality tool.

Test quality tool link:

iPhone 6+(12.5.4):

https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469280
Verified on Devices: iPhone 6+(12.5.4)
CC: @gnardini
Thanks..!!

**Comment by ValoraQA** [comment]
Hey MuckT We have verified this issue on Test flight Alfajores build V1.18.0 and updated the status in test quality tool.

**Test Quality Tool:**
- https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469269
- https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469274

**Verified On Device:** iPhone SE (13.6.1)

CC: @gnardini 
Thanks..!!

**Comment by ellenwoconnor** [comment]
any ideas why this shows up in new issues even though it's closed? how can we get rid of it?

**Pull request merged**